src/packages/core/components/icon/icon.element.ts

```diff
@@ -48,7 +72,9 @@ export class UmbIconElement extends UmbLitElement {
 	}
 
 	override render() {
-		return html`<uui-icon name=${ifDefined(this._icon)} style=${ifDefined(this._color)}></uui-icon>`;
+		return html`<uui-icon
+			name=${ifDefined(this._icon)}
+			style=${ifDefined(this._color ?? this._fallbackColor)}></uui-icon>`;
```

Comment:
Do you think we could use [styleMap](https://lit.dev/playground/#sample=examples/directive-style-map) to avoid some of the extra logic around calculating a CSS variable?

Reply:
Just pushed a commit using stylemap now